### ISSUE TITLE
TypeError: postcss_1.default is not a function

### ISSUE THREAD

**Issue by ulivz** [vuejs/component-compiler-utils#9]
# Summary

Version: 1.2.0

Recently many users reported an error at https://github.com/vuejs/vuepress/issues/268, and I reproduced it at my local.

After the investigation, I found:

https://github.com/vuejs/component-compiler-utils/blob/14d4ced9bc7ac4514bcbf51ca133529088c2887b/lib/compileStyle.ts#L70

It's will be compiled into:

```js
result = postcss_1.default(plugins).process(source, postCSSOptions);
```

But it postcss's dist code, the module was exported as:

```js
exports.default = postcss;
module.exports = exports['default'];
```

So the module should be used as `postcss_1` instead of ` postcss_1.default`.

And I don't know if this is a problem of TSC. but it's truly a issue introduced by upgrade of `vue-loader`/`@vuejs/component-compiler-utils`

**Comment by bayssmekanique** [comment]
I noticed that https://github.com/vuejs/component-compiler-utils/commit/1040f0accff912f036012390acce743b078ae904 added a `rm` to the task, but depending on the build system `rm` might not be a valid command (atleast it's not on Windows).

**Comment by 719media** [comment]
This happened to me as well.
Compiled js before (version 1.1.0)
`result = postcss(plugins).process(source, postCSSOptions);`

Compiled js after (version 1.2.0):
`result = postcss_1.default(plugins).process(source, postCSSOptions);`

**Comment by znck** [comment]
Fixed in `1.2.1`.
